test/util/mocks/request_processor.mocks.dart

```diff
@@ -18,8 +18,7 @@ import 'request_processor.dart' as _i2;
 // ignore_for_file: camel_case_types
 // ignore_for_file: subtype_of_sealed_class
 
-class _FakeProcessorResult_0 extends _i1.SmartFake
-    implements _i2.ProcessorResult {
+class _FakeProcessorResult_0 extends _i1.SmartFake implements _i2.ProcessorResult {
```

Comment:
I assume this change happened when you formatted the project? I would revert it because it's an autogenerated file, not our concern/responsibility

Reply:
how do I revert that? If I save it formats it back all the time

Reply:
Sorry I should have changed the README. It's
`flutter format --line-length=120 . $(find lib test -name "*.dart" -not -name "*.mocks.dart")`